usr/src/test/smartos-test/smartos-test.sh

```diff
@@ -489,7 +458,7 @@ if [[ $do_configure = true ]]; then
     add_loopback_mounts $test_archive
     extract_remaining_test_bits $test_archive
     add_test_accounts
-    setup_pkgsrc
+    /smartdc/bin/pkgsrc-setup
```

Comment:
I was curious to know whether there was a possibility that the exit status could change here depending on whether pkgsrc tools had already been installed, but the script explicitly exit's 0 so I think we're fine, the only meaningful change being the slightly different output text.